ext/dom/php_dom.stub.php

```diff
@@ -99,25 +99,25 @@ class DOMNameSpaceNode
 
 class DOMImplementation
 {
-    /** @return void */
-    public function getFeature(string $feature, string $version) {}
+    /** @tentative-return-type */
+    public function getFeature(string $feature, string $version): void {}
```

Comment:
Not familiar with this functionality, but noticed randomly: I think `never` would be more appropriate, but because `never` is added in 8.1, it would (somewhat) prevent overriding this in everything older than 8.1

Then again, https://developer.mozilla.org/en-US/docs/Web/API/DOMImplementation mentions that no browser ever implemented getFeature and it was removed from the spec, so this actually makes sense, but documenting the reason for a getter returning void (in c file and .stub.php file) would avoid confusion.

https://www.php.net/domimplementation mentions the method is undocumented

Maybe keep it as `: void` and add `@return never`

Aside: `Not implemented` may make more sense than`"Not yet implemented"` but this is way out of scope.

```c
PHP_METHOD(DOMImplementation, getFeature)
{
	size_t feature_len, version_len;
	char *feature, *version;

	if (zend_parse_parameters(ZEND_NUM_ARGS(), "ss", &feature, &feature_len, &version, &version_len) == FAILURE) {
		RETURN_THROWS();
	}

	DOM_NOT_IMPLEMENTED(); // macro for zend_throw_error(NULL, "Not yet implemented");
}
```

Reply:
I've just gone with `never`